From 9e175fc144b65e4d28026e88eba49a15235d2444 Mon Sep 17 00:00:00 2001
From: Sergio Lopez <slp@redhat.com>
Date: Tue, 24 Jun 2025 10:54:04 +0000
Subject: [PATCH] virtio/vulkan: silence stuck in wait message (HACK)

---
 src/virtio/vulkan/vn_common.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/virtio/vulkan/vn_common.c b/src/virtio/vulkan/vn_common.c
index 0fb0312c..ee9ea031 100644
--- a/src/virtio/vulkan/vn_common.c
+++ b/src/virtio/vulkan/vn_common.c
@@ -262,8 +262,8 @@ vn_relax(struct vn_relax_state *state)
 
    if (unlikely(*iter % (1 << warn_order) == 0)) {
       struct vn_instance *instance = state->instance;
-      vn_log(instance, "stuck in %s wait with iter at %d", state->reason_str,
-             *iter);
+      //vn_log(instance, "stuck in %s wait with iter at %d", state->reason_str,
+      //       *iter);
 
       struct vn_ring *ring = instance->ring.ring;
       const uint32_t status = vn_ring_load_status(ring);
-- 
2.47.1

